notebooks/02-technical-tutorials/virtual-hierarchies/main.ipynb

```diff
@@ -0,0 +1,852 @@
+      ],
+      "text/plain": [
+       "Open the notebook in JupyterLab with the Atoti extension enabled to build this widget."
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "session.visualize(\"Virtual Hierarchy\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3411ae4e-d804-4c11-8930-e1e523c25128",
+   "metadata": {},
+   "source": [
+    "### MDX with Non Empty for Virtual Hierarchy \n",
+    "\n",
+    "In [Multidimensional Expressions (MDX)](https://learn.microsoft.com/en-us/analysis-services/multidimensional-models/mdx/mdx-query-the-basic-query?view=asallproducts-allversions), the keyword [`Non Empty`](https://learn.microsoft.com/en-us/sql/mdx/nonempty-mdx?view=sql-server-ver16) is used to filter out empty cells from the result set of a query. Since we are not storing the members of the virtual hierarchy in the cube, the query will not be able to return any results for virtual hierarchies. Let's see this with an example:    \n",
```

Comment:
I removed the attempt to explain why it cannot be done.
Instead, I just show them the error message. 

Do you think this works? Otherwise, maybe we can just remove the section.